content/blog/2016/2016-12-19-declarative-pipeline-beta.adoc

```diff
@@ -8,7 +8,7 @@ author: hrmpw
 ---
 
 Last week we released version 0.7.1 of the
-link:https://wiki.jenkins.io/display/JENKINS/Pipeline+Model+Definition+Plugin[_Pipeline-Model-Defintion_]
+link:/doc/book/pipeline/[_Pipeline-Model-Definition_]
```

Comment:
Specifically refers to the plugin, so https://plugins.jenkins.io/pipeline-model-definition/ would be better.